Podfile

```diff
@@ -6,16 +6,13 @@ use_frameworks!
 platform :osx, '10.13'
 workspace 'Simplenote.xcworkspace'
 
-plugin 'cocoapods-repo-update'
-
-
 # Main
 #
 abstract_target 'Automattic' do
 
   # Automattic Shared
   #
-  pod 'Automattic-Tracks-iOS', '~> 0.6'
+  pod 'Automattic-Tracks-iOS', git: 'git@github.com:Automattic/Automattic-Tracks-iOS', branch: 'update/sentry-to-v6'
```

Comment:
**It would be appropriate to wait till an official version of Tracks has been released before merging this PR.**

That might mean delaying shipping version 2.6 to the users for a few days, or longer in case there are issues with the Sentry migration. 

Alternatively, we can shift the Apple Silicon support to version 2.7 and have a more thorough testing of it.

Reply:
Good on my end! We can switch when it's available!

Reply:
I've switched this over